### ISSUE TITLE
Support for markdown package

### ISSUE THREAD

**Issue by kieran-mace** [tjkohli/nebula-syntax#8]
Can you add support for the language-markdown package located [here](https://github.com/burodepeper/language-markdown)?

**Comment by tjkohli** [comment]
Hi @kmace, thanks for the post! I'll consider it. As always, anyone is free to contribute to this as well.

**Comment by kieran-mace** [comment]
Thank you!

I have no idea how to add support, but I'd be willing to help if I can get some guidance.

**Comment by sdrakulich** [comment]
I'd like to reboot this. I love this theme, it's my favorite and the one I want to be using.
However, any function in R is blinking and it makes me sad. I'd fix it but I don't know how.

**Comment by tjkohli** [comment]
@sdrakulich that doesn't sound like an issue related to this thread. Can you please create a new issue and maybe I can address it there?

**Comment by tjkohli** [comment]
@sdrakulich can you please send me a sample R file where this happens? I will try to see why it is happening.

**Comment by kieran-mace** [comment]
here is an example:

~~~ markdown
---
title: "R Notebook"
output: html_notebook
---

Simple example to show rmarkdown weirdness.

```{r}
library(dplyr)
library(downloader)
url <- "https://raw.githubusercontent.com/genomicsclass/dagdata/master/inst/extdata/msleep_ggplot2.csv"
filename <- "msleep_ggplot2.csv"
if (!file.exists(filename)) download(url,filename)
msleep <- read.csv("msleep_ggplot2.csv")
head(msleep)
sleepData <- select(msleep, name, sleep_total)
head(sleepData)
```
~~~ 

save that as `example.Rmd`

**Comment by tjkohli** [comment]
@kmace Unfortunately, this confirms my suspicions. Atom and/or the language-markdown does not seem to support syntax highlighting for R. It doesn't differentiate logic and, therefore, there is no way for my theme to style it. I actually tested this file with both default Atom themes: Atom Dark and One Dark (both UI & Syntax), and neither of them highlight your file any better than Nebula does.
 
<img width="1046" alt="screen shot 2018-05-08 at 7 08 46 pm" src="https://user-images.githubusercontent.com/7361247/39787457-82780692-52f3-11e8-9902-3aab3e5c22af.png">

**Comment by tjkohli** [comment]
@sdrakulich can you please provide me your sample file that highlights the blinking function? I was not able to reproduce it.

**Comment by kieran-mace** [comment]
@tjkohli, you are correct, but if you also install the language-markdown here: https://github.com/burodepeper/language-markdown

you get the following:

![screen shot 2018-05-08 at 4 23 26 pm](https://user-images.githubusercontent.com/18296227/39787853-503aa9f8-52dc-11e8-9747-d6f19efff732.png)

**Comment by kieran-mace** [comment]
language-markdown has support for different themes, with instructions listed here:
https://github.com/burodepeper/language-markdown/blob/master/CONTRIBUTING.md#syntax-theme-support

**Comment by tjkohli** [comment]
@kmace I have had the language-markdown installed. Just installed minimal-syntax theme, and doesn't seem like it's highlighting your logic either. Perhaps there is something wrong with my Atom install.

<img width="809" alt="screen shot 2018-05-08 at 8 44 26 pm" src="https://user-images.githubusercontent.com/7361247/39789937-e709e398-5300-11e8-8c3d-875005ef5f93.png">

Just to be clear, you have an issue with the error underlining in .Rmd, am I correct?
